Mathlib/Order/Cofinal.lean

```diff
@@ -0,0 +1,102 @@
+/-
+Copyright (c) 2024 Violeta Hernández Palacios. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Violeta Hernández Palacios
+-/
+import Mathlib.Order.Bounds.Basic
+
+/-!
+# Cofinal sets
+
+A set `s` is cofinal when for every `a` there exists an element of `s` greater or equal to it. This
```

Comment:
A set in what kind of type? Maybe add `Let alpha be a type with [LE alpha]`.

Reply:
I don't think you should be bombarding your reader with Lean notation from the first sentence of the docstring. What about
```suggestion
A set `s` in an order is cofinal when for every `a` there exists an element of `s` greater or equal to it. This
```

Reply:
“in an ordered type” seems fine to me. At least the reader has to be told what are the implicit datas used in the definition, and not to guess depending on the future syntax. 
(And “bombarded” is of the passive-aggressive kind of vocabulary that should be avoided in written discussion.)

Reply:
I've rewritten this using the "ordered type" wording.